docs/installation/manual/installing-solr.md

```diff
@@ -66,33 +67,12 @@ Then apply your new configuration.
 sudo sysctl -p
 ```
 
-### Creating a New Solr Core
-
-Initially, our new Solr core will contain a configuration copied from the example included with the installation, so that we have something to work with when we configure this on the Drupal side. We’ll later update this with generated configurations we create in Drupal.
-
-```bash
-cd /opt/solr
-sudo mkdir -p /var/solr/data/SOLR_CORE/conf
-sudo cp -r example/files/conf/* /var/solr/data/SOLR_CORE/conf
-sudo chown -R solr:solr /var/solr
-sudo -u solr bin/solr create -c SOLR_CORE -p 8983
-```
-- `SOLR_CORE`: `islandora8`
-
-You should see an output similar to this:
-```
-WARNING: Using _default configset with data driven schema functionality. NOT RECOMMENDED for production use.
-         To turn off: bin/solr config -c islandora8 -p 8983 -action set-user-property -property update.autoCreateFields -value false
-
-Created new core 'islandora8'
-```
```

Comment:
Why was this removed? The rest of the page refers to "the core we created earlier" but there is no core now

Reply:
Update: it looks like this was just moved down to line 134. IMO this needs to be updated to be in the right order. I can make this change without objection

Reply:
@forresthopkinsa , this PR has been merged already, but if you would like to propose any edits, you can do it on https://github.com/Islandora/documentation/blob/main/docs/installation/manual/installing-solr.md